scriptmodules/supplementary/esthemes.sh

```diff
@@ -51,7 +51,8 @@ function gui_esthemes() {
         'ehettervik pixel-metadata'
         'ehettervik pixel-tft'
         'ehettervik luminous'
-        'ehettervik minilumi'
+
+'ehettervik minilumi'
```

Comment:
Oops sorry I have no idea how I messed that up wasnt even close to where I added my line.